.travis.yml

```diff
@@ -65,9 +65,11 @@ matrix:
 # Validate rule files
         - make validateRules
 # check htmlreport stuff
-        - ./htmlreport/test_htmlreport.py
+        - python2 htmlreport/test_htmlreport.py
+        - python3 htmlreport/test_htmlreport.py
```

Comment:
as far as I see, `test_htmlreport.py` executes `htmlreport.py` internally and I guess it does not use python3 then.

    subprocess.check_call(
        [HTML_REPORT_BIN,
         '--file=' + os.path.realpath(xml_filename),
         '--report-dir=' + os.path.realpath(output_directory)],
        cwd=os.path.join(ROOT_DIR, 'htmlreport'))

Reply:
Oops, yes. sorry, my bad. Fixed.